platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy

```diff
@@ -0,0 +1,135 @@
+package platform
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.GENERAL_PRIVATE_JOB_SECURITY
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.GHPRB_CANCEL_BUILDS_ON_UPDATE
+
+/* stdout logger */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+/* Map to hold the k:v pairs parsed from the secret file */
+Map ghprbMap = [:]
+try {
+    out.println('Parsing secret YAML file')
+    String ghprbConfigContents = new File("${GHPRB_SECRET}").text
+    Yaml yaml = new Yaml()
+    ghprbMap = yaml.load(ghprbConfigContents)
+    out.println('Successfully parsed secret YAML file')
+}
+catch (any) {
+    out.println('Jenkins DSL: Error parsing secret YAML file')
+    out.println('Exiting with error code 1')
+    return 1
+}
+
+Map publicBokchoyJobConfig = [
+    open: true,
+    jobName: 'edx-platform-bokchoy-pipeline-pr',
+    repoName: 'edx-platform',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/bokchoy-wtw',
+    onlyTriggerPhrase: true,
+    triggerPhrase: /.*jenkins\W+run\W+bokchoy\W+wtw.*/,
```

Comment:
I believe these will also match to the regular expressions in the normal PR job trigger phrases. You should make them more unique, like "jenkins run wtw bokchoy" or similar.

Reply:
Good call.